prometheus/utils.go

```diff
@@ -26,6 +27,8 @@ func connectRaw(ctx context.Context, cc *connection.ConnectionCache, c *plugin.C
 
 	// Prefer config settings
 	prometheusConfig := GetConfig(c)
+
+	address := os.Getenv("PROMETHEUS_URL")
```

Comment:
@beudbeud The lint check indicates an invalid address assignment, were you able to compile + run this code?

As a suggestion, you could probably remove the declaration on line 26:
```go
var address string
```